Update pg_to_airtable for analysis.hoa_segments schema change

mcchang opened the pull request:
`analysis.hoa_segments` is now an integer from 0-4 instead of a string

mcchang commented:
@pcothenet can you invite me to `climatechangemakers` sls app?

pcothenet commented:
Absolutely. Right now it's on my personal account so I want to spend a
little time this weekend to move them to a separate org, and then I can
share my login. I'm happy to spend a little time deploying your change in
the meantime.

I'll also set things to auto-deploy on commit to master.

On Tue, Mar 30, 2021 at 7:12 PM Michael Chang ***@***.***>
wrote:

> @pcothenet <https://github.com/pcothenet> can you invite me to
> climatechangemakers sls app?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/climatechangemakers/airtable_sync/pull/12#issuecomment-810658598>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALEOJUCE5GOVLVD66TE5ULTGJSGBANCNFSM42DILL4A>
> .
>

pcothenet reviewed (commented):
Looks good, but if I try to run it, I get a `error: permission denied for view hoa_segments`

mcchang commented:
> Looks good, but if I try to run it, I get a `error: permission denied for view hoa_segments`

Ok I think I fixed it now.  Can you give it another shot?
![image](https://user-images.githubusercontent.com/66332/113325412-3d571400-92cd-11eb-8e36-989dcefad091.png)

pcothenet commented:
It works now. I fixed a tiny bug and will deploy it. I'm setting it to run
every 24 hours for now.

On Thu, Apr 1, 2021 at 9:33 AM Michael Chang ***@***.***>
wrote:

> Looks good, but if I try to run it, I get a error: permission denied for
> view hoa_segments
>
> Ok I think I fixed it now. Can you give it another shot?
> [image: image]
> <https://user-images.githubusercontent.com/66332/113325412-3d571400-92cd-11eb-8e36-989dcefad091.png>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/climatechangemakers/airtable_sync/pull/12#issuecomment-812026806>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALEOJRZ7XFPV4VLFUOQVT3TGSN45ANCNFSM42DILL4A>
> .
>

mcchang commented:
> It works now. I fixed a tiny bug and will deploy it. I'm setting it to run every 24 hours for now.
> […](#)
> On Thu, Apr 1, 2021 at 9:33 AM Michael Chang ***@***.***> wrote: Looks good, but if I try to run it, I get a error: permission denied for view hoa_segments Ok I think I fixed it now. Can you give it another shot? [image: image] <https://user-images.githubusercontent.com/66332/113325412-3d571400-92cd-11eb-8e36-989dcefad091.png> — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub <[#12 (comment)](https://github.com/climatechangemakers/airtable_sync/pull/12#issuecomment-812026806)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AALEOJRZ7XFPV4VLFUOQVT3TGSN45ANCNFSM42DILL4A> .

Thank you @pcothenet !
